source/security.txt

```diff
@@ -7,13 +7,63 @@ Security
 
 .. default-domain:: mongodb
 
-Use the following resources to learn about security in {+service+}:
+The Security pillar of the {+service+} {+waf+} includes features that
+ensure data integrity and privacy through comprehensive safeguarding
+measures, such as encryption at rest and in transit, fine-grained access
+controls, and adherence to regulatory standards.
 
-- :ref:`Network Security <arch-center-network-security>`
-- :ref:`Authorization and Authentication <arch-center-auth>`
-- :ref:`Data Encryption <arch-center-data-encryption>`
-- :ref:`Compliance <arch-center-compliance>`
-- :ref:`Auditing and Logging <arch-center-auditing-logging>`
+Use the following {+atlas-arch-center+} resources to learn about
+security in {+service+}:
+
+.. card-group::
+   :columns: 2
+   :style: extra-compact
+
+   .. card::
+      :headline: Network Security
+      :url: https://mongodb.com/docs/manual/data-modeling/
+      :icon: general_security
+      :icon-alt: Atlas security icon
+
+      |service-fullname| enforces mandatory |tls| encryption for database connections.
+      Restrict database access to specific IP addresses and |cidr| ranges
+      and use private endpoints for staging and production projects.
```

Comment:
Let's remove the first line here to keep the cards ~4 lines each for consistency of appearance
```suggestion
      Restrict database access to specific IP addresses and |cidr| ranges
      and use private endpoints and VPC peering.
```